.ibm/pipelines/resources/postgres-db/postgres.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PostgresCluster
+metadata:
+  name: postgress-external-db
+  namespace: postgress-external-db
+spec:
+  image: registry.developers.crunchydata.com/crunchydata/crunchy-postgres:ubi8-14.2-1
+  postgresVersion: 14
```

Comment:
@subhashkhileri Can we change it to use version 15.1? It would be good to use the same postgres version that RHDH 1.2 supports

Reply:
Updated to version 16.